presto-password-authenticators/src/main/java/io/prestosql/plugin/password/ldap/LdapAuthenticator.java

```diff
@@ -55,6 +56,7 @@
         implements PasswordAuthenticator
 {
     private static final Logger log = Logger.get(LdapAuthenticator.class);
+    private static final String SPECIAL_LDAP_CHARACTERS = ",=+<>#;*()\"\\\u0000";
```

Comment:
Use `com.google.common.base.CharMatcher#anyOf`

Then use `CharMatcher#matchesAnyOf` for checking if rogue char is present